source/initial_temperature/adiabatic_boundary.cc

```diff
@@ -187,13 +117,18 @@ namespace aspect
                                               "adiabatic boundary",
                                               "An initial temperature condition that allows for discretizing "
                                               "the model domain into two layers separated by a user-defined "
-                                              "isothermal boundary using a table look-up approach. The user includes an "
-                                              "input ascii data file that is formatted as 3 columns of `latitude', "
-                                              "`longitude', and `depth', where `depth' is in kilometers and "
-                                              "represents the depth of an initial temperature of 1673.15 K (by default). "
+                                              "isothermal boundary. The user includes an input ascii data file "
+                                              "that is formatted as 3 columns of `longitude(radians)', "
+                                              "`colatitude(radians)', and `depth(meters)', where `depth' represents the depth "
+                                              "of an initial temperature of 1673.15 K (by default). "
+                                              "The first lines in the data file may contain any number of comments if they begin "
+                                              "with `#', but one of these lines needs to contain the number of grid points "
+                                              "in each dimension as for example `# POINTS: 69 121'. Note that the data need "
+                                              "to be sorted in a specific order: the `longitude' coordinate needs to ascend first, "
+                                              "followed by the `colatitude' coordinate, and then the `depth' coordinate at last "
+                                              "in order to assign the correct data to the prescribed coordinates. "
                                               "The temperature is defined from the surface (273.15 K) to the isotherm "
                                               "as a linear gradient. Below the isotherm the temperature increases "
-                                              "approximately adiabatically (0.0005 K per meter). This initial temperature condition "
-                                              "is designed specifically for the ellipsoidal chunk geometry model.")
+                                              "approximately adiabatically (0.0005 K per meter).")
```

Comment:
The AsciiData class should work for cartesian models as well, but currently we have no test for that. Please either create a test (best option), or make sure the plugin tests ensures that we have a spherical geometry model by adding an `AssertThrow` (like you had before for the ellipsoidal model), or add the following text to the documentation:

```suggestion
                                              "approximately adiabatically (0.0005 K per meter). "
                                              "This plugin should work for all geometry models, but is currently only tested for spherical "
                                              models.")
```